mlfinlab/bet_sizing/test_ch10_snippets.py

```diff
@@ -0,0 +1,104 @@
+"""
+Tests the unit functions in ch10_snippets.py for calculating bet size.
+"""
+
+import unittest
+import datetime as dt
+import numpy as np
+import pandas as pd
+
+from scipy.stats import norm
+
+from mlfinlab.bet_sizing.ch10_snippets import (get_signal, avg_active_signals,

```

Comment:
Remove `()`

Reply:
Same regarding parentheses as previous. Let me know if `\` or multiple `from aaa.bbb import xx, yy, zz` lines are preferred.

Reply:
As per prev comment, I agree with your Pep 328 style with the (). Let's keep using that.